encryption_works.md

```diff
@@ -299,6 +281,16 @@ Here's a message that was encrypted to my public key. Without having access to m
     =a8FR
     -----END PGP MESSAGE-----
 
+
+### The Web of Trust 
+
+PGP works best when a web of trust is created. Strong crypto can still be broken since humans are the weakest elements of any security system. People using PGP can fall victim to what's called a "man in the middle attack." Let's say you're talking to your colleague via encrypted e-mail. An adversity could potentially trick you into thinking that their public key was your colleagues. You'd encrypt a message with your private key and my public key. I can decrypt it, read and/or tamper with it, and then send the compromised message onto your college, with info purported to be from you. 
+
+This can be solved, or at least mitigated, if you and your colleague independently verify each other's keys. This way, you verify your colleague's public key independently of when they e-mail you. But this raises another problem -- meeting people and verifying their keys is time-consuming. 
+
+The answer to this is the web of trust. As time goes on, you meet people and verify their keys. If you sign those keys (publicly asserting that you've verified them), that functions as a vouch of sorts -- this key has verified that key belongs to who it says it does. If I trust your key, by extension I should trust the keys you've signed, even if I don't know the signed key personally. I trust it because I trust you.
```

Comment:
maybe this is a nit-pick, but people should be choosy about their WoT.  There are a billion people I know whose keys I don't sign, and I don't want them to sign/upload mine.  Because we wouldn't be comfortable with the whole world knowing that we communicate with PGP.

Journalists should be especially attuned to this issue, especially when sources are involved.